nf_core/components/install.py

```diff
@@ -58,6 +58,9 @@ def install(self, component, silent=False):
         if not silent:
             modules_json.check_up_to_date()
 
+        # Verify that the remote repo's org_path does not match the org_path of any alternate repo among the installed modules
+        self.check_alternate_remotes(modules_json)
```

Comment:
This won't make the command fail, I would rather throw an error if this function returns true.